### ISSUE TITLE
Toggling between colors and background

### ISSUE THREAD

**Issue by Chauncey1989** [ga-dc/project1#177]
I am having trouble toggling between colors and the background.

**Comment by RobertAKARobin** [comment]
Can you be more specific? What have you tried? What code are you using? What errors are you getting? 

Please refer to this: https://github.com/ga-dc/wdi7/blob/master/asking-for-help.md

**Comment by Chauncey1989** [comment]
I tried `$( ".Card1" ).toggleClass( ".Card1", addOrRemove );`.
I expected the background to toggle out and the new color to appear.
nothing happened instead.

I have tried looking up solutions on http://api.jquery.com/toggleclass/ and I have not been successful in finding an answer.

**Comment by RobertAKARobin** [comment]
I think the problem is you have too much stuff in your `toggleClass`. There should only be one thing in the parentheses. Also, it _doesn't_ have to have a period because the fact that it's a class is inferred from the word "class" in "toggleClass".

TLDR: Try this:

``` js
$(".Card1").toggleClass("Card1");
```

**Comment by Chauncey1989** [comment]
I tried get rid of all the non matches in the memory game.
I expected the non matches to hide.
the non matches stay up indead happened instead.
